plugins/Marketplace/vue/src/PluginList/PluginList.vue

```diff
@@ -71,6 +71,12 @@
                 <div class="card-description">{{ plugin.description }}</div>
               </div>
               <div class="card-content-bottom">
+                <div class="owner">
+                  {{translate('Marketplace_CreatedBy')}}
+                  <span
+                    v-if="plugin.owner === 'piwik' || plugin.owner === 'matomo-org'"> Matomo</span>
+                  <span v-else> {{plugin.owner}}</span>
```

Comment:
feels better readable with spaces
```suggestion
                  <span v-else> {{ plugin.owner }}</span>
```

Reply:
done :+1: